azkaban-web-server/src/main/java/cloudflow/models/ExecutionBasicResponse.java

```diff
@@ -3,16 +3,17 @@
 import static java.util.Objects.requireNonNull;
 
 import azkaban.executor.ExecutableFlow;
+import azkaban.executor.ExecutionOptions.FailureAction;
 
 
 public class ExecutionBasicResponse {
   private String executionId;
   private String submitUser;
   private long submitTime;
   private String concurrentOption;
-  private String failureAction;
-  private boolean isNotifyFailureFirst;
-  private boolean isNotifyFailureLast;
+  private FailureAction failureAction;
```

Comment:
Thanks for making this field an enum. 
Now we need to use a Jackson annotation (if available) to specify the method we want to use to serialize it (the default is the enum value but we want a string associated with it).